lucien-ios/AddAndEditNewComicBook/ComicFormViewController.swift

```diff
@@ -59,6 +59,10 @@ final class ComicFormViewController: UIViewController, AlertDisplaying {
     private let cameraViewController = CameraViewController()
     private let disposeBag = DisposeBag()
 
+    // MARK: - Variables
+
+    var delegate: DismissViewController?
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Weak Delegate Violation: Delegates should be weak to avoid reference cycles. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Aweak_delegate)

Reply:
Let's rename `DismissViewController` to `ViewControllerDismissingDelegate` . `DismissViewController` sounds  like an actual `ViewController` that might exist in our codebase.